### DISCUSSION TITLE
Does the emulator support adding entities on fly?

### DISCUSSION THREAD

**Discussion by GabrielHSFerreira** [Azure/azure-service-bus-emulator-installer#23] [Q&A] [answered]
Does the emulator support adding new entities (topics, subscriptions, queues) during execution? Or only via the config.json file?

**Comment by krishankumar95** [comment, marked as answer]
@GabrielHSFerreira ; In the current release following features are not supported:
a) AdminClient for Entity Management operations on the fly. Entities can only be created via Config.json on bootup. 
b) SQL Filters and Actions
c) Partitioned Entities
